scripts/zones/RuLude_Gardens/npcs/Delivery_Crate.lua

```diff
@@ -7,18 +7,18 @@
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
-    xi.magian.deliveryCrateOnTrade(player, npc, trade)
+    xi.magian.deliveryCrateOnTrade(player, trade)
 end
 
 entity.onTrigger = function(player, npc)
 end
 
 entity.onEventUpdate = function(player, csid, option, npc)
-    xi.magian.deliveryCrateOnEventUpdate(player, csid, option, npc)
+    xi.magian.deliveryCrateOnEventUpdate(player, csid, option)
 end
 
 entity.onEventFinish = function(player, csid, option, npc)
-    xi.magian.deliveryCrateOnEventFinish(player, csid, option, npc)
+    xi.magian.deliveryCrateOnEventFinish(player, csid, option)
```

Comment:
I don't know what the others think, but I prefer the pattern where we just forward _all_ the args into the global function, instead of picking and choosing which ones we want.

So the change here would be going into xi.magian functions and making them take the npc as an argument, even if it isn't going to use it

Reply:
I have no strong feelings one way or another. So whatever you guys decide works for me

Reply:
I'm going to agree with Zach on this one.  Even though its unused, its one of those functions that I like seeing them all present

Reply:
kk. Addressed